docs/developer/modules.rst

```diff
@@ -0,0 +1,16 @@
+Modules
+-------
+
+.. toctree::
+
+   modules/geomesa-compute
```

Comment:
i would double-check that the text in all these modules is up-to-date with the readmes - there have been a lot of changes recently.